src/test/java/org/openmrs/contrib/qaframework/test/FormTest.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.test;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.contrib.qaframework.helper.BuildTests;
+import org.openmrs.contrib.qaframework.helper.TestBase;
+import org.openmrs.contrib.qaframework.page.AdministrationPage;
+import org.openmrs.contrib.qaframework.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.contrib.qaframework.page.HomePage;
+import org.openmrs.contrib.qaframework.page.HtmlFormsPage;
+import org.openmrs.contrib.qaframework.page.ManageFormsPage;
+import org.openmrs.contrib.qaframework.page.ManageHtmlFormsPage;
+import org.openqa.selenium.By;
+
+/**
+ * Created by sharif
+ */
```

Comment:
any reason for adding comment?

Reply:
Probably for referring author of the class.